src/Engine/Physics/Trigger.cpp

```diff
@@ -25,7 +24,7 @@ namespace Physics {
         return 0; // Was a planned purpose, but is not used.
     }
 
-    btCollisionObject* Trigger::GetCollisionObject() {
+    btCollisionObject* Trigger::GetCollisionObject() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetCollisionObject' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)